ee/tabby-webserver/src/service/background_job/mod.rs

```diff
@@ -19,13 +21,11 @@ use tabby_schema::{
     repository::{GitRepositoryService, ThirdPartyRepositoryService},
     web_crawler::WebCrawlerService,
 };
-use third_party_integration::SchedulerGithubGitlabJob;
+pub use third_party_integration::SchedulerGithubGitlabJob;
 use tracing::warn;
-use web_crawler::WebCrawlerJob;
+pub use web_crawler::WebCrawlerJob;
```

Comment:
instead of exposing each job by their name, i recommend just exposing methods like below in service:
```
get_latest_web_crawler_job(param: String)
```